### DISCUSSION TITLE
Patterns and real world projects

### DISCUSSION THREAD

**Discussion by mustafa0x** [vitalik/django-ninja#516] [General]
Hi, I'm finding django ninja very appealing, thanks!

The documentation is great, and the nice thing about ninja is that it's a minimalistic framework.

With that said, it would be useful to look at idiomatic patterns, especially from real world projects. I haven't found that many however, at most 1-2.

**Comment by SmileyChris** [comment]
Hey, you're totally right. The documentation structure is changing (check out the main branch and you can build them locally) and having some tutorials based on simple real world projects would be good to add. If you have specific suggestions (or better still, can start writing the bones of it) then please share!
